Sources/protoc-gen-swift/ExtensionGenerator.swift

```diff
@@ -19,6 +19,101 @@ import PluginLibrary
 import SwiftProtobuf
 
 class ExtensionGenerator {
+
+    // Helper to hold all the extensions when generating things across all
+    // the extensions within a given file.
+    class Set {
```

Comment:
This should probably be renamed so it doesn't ever risk colliding with the stdlib type of the same name (e.g., if someone wanted to use Swift's `Set` elsewhere in the `ExtensionGenerator` scope).